### PR TITLE
React capstone project - Metrics webapp

### PR DESCRIPTION
**Pull request by ntzwilly**
## Description:
- Created React components (Home and Details pages)
- Used React Router.
- Connected React and Redux.
- Wrote integration tests with a React testing library.
- Added styles.
- Used React life cycle methods.
- Used store, actions, and reducers in React.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,64 @@
...
+
+### Install
+
+    npm install
+
+### Usage
+
+    npm start
+
+## Authors
```
Comment (rloterh) [inline README.md]: - [x] To improve the professionalism aspect of your ReadMe, kindly change the header title _Authors_ to _Author_, as this project is done solely.

File: README.md [hunk 2]
```diff
@@ -1 +1,64 @@
-# metrics-app
\ No newline at end of file
+# metrics-app
+
```
Comment (rloterh) [inline README.md:2]: Your _ReadMe_ needs some improvements so as to make it appear more professional. Kindly note that _ReadMe_ is mostly the first place that visitors of this project's repository will go and therefore it has to be very detailed and professional. Also, it needs to be done well with respect to this project,  which will likely feature on your portfolio. Kindly ensure to improve upon these aspects of your ReadMe. 
- [x] Project description, how to set up, run and test this project should be detailed, clear, and well organized.
- [x] The code aspects in each procedure should be in backticks`.

File: src/redux/countries/countries.js [hunk 3]
```diff
@@ -0,0 +1,82 @@
+/* eslint-disable no-case-declarations */
```
Comment (rloterh) [inline src/redux/countries/countries.js]: - [x] You have **disabled linter check** on your code. Please enable linter check on your code to ensure code quality.
- [x] Test case is failing. Kindly ensure to correct import statements and all test cases are passing.

**Failing Test case:**
image

File: src/components/Home.js [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+import { HiMicrophone } from 'react-icons/hi';
+import { MdSettings } from 'react-icons/md';
+import { BsArrowRightCircle } from 'react-icons/bs';
+import { useSelector, useDispatch } from 'react-redux';
+import { fetchCountries } from '../redux/countries/countries';
+import europe from '../assets/maps/europe.svg';
+import covid from '../assets/icons/virus.svg';
+import '../css/Home.css';
+
+const Home = () => {
```
Comment (rloterh) [inline src/components/Home.js:13]: - [x] As specified, kindly implement a **redux filter/ search** functionality so as to enable filtering of countries. The filter/ search field should be displayed on the Homepage.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Willy, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to complete this project but you are almost there! 🚦  🚦  

## Good Points 👍 
- Descriptive Pull Request
- App is correctly deployed
- Good video presentation of the project
- Design matches provided template
- Redux is effectively used to manage state data
- Test cases are passing

## Required Changes ♻️
_Kindly check the comments under the review._

<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 4 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (ntzwilly) [reply README.md]: Thank you. I have made the change

File: README.md (hunk 2 above)
Comment (ntzwilly) [reply README.md:2]: It is done. Thank you

File: src/redux/countries/countries.js (hunk 3 above)
Comment (ntzwilly) [reply src/redux/countries/countries.js]: Thank you for the feedback. I have unable the linter check and all tests are passing now

File: src/components/Home.js (hunk 4 above)
Comment (ntzwilly) [reply src/components/Home.js:13]: This now is done.

File: README.md (hunk 1 above)
Comment (peterrobert) [reply README.md]: This change has been done well

File: README.md (hunk 2 above)
Comment (peterrobert) [reply README.md:2]: This change has been done well

File: src/redux/countries/countries.js (hunk 3 above)
Comment (peterrobert) [reply src/redux/countries/countries.js]: This change has been done well

File: src/components/Home.js (hunk 4 above)
Comment (peterrobert) [reply src/components/Home.js:13]: This change has been implemented well

**Review by peterrobert — APPROVED** [review]
Hi @ntzwilly, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**